common/app/templates/inlineJS/blocking/cloudwatchBeacons.scala.js

```diff
@@ -1,6 +1,18 @@
 @(item: model.MetaData)(implicit request: RequestHeader)
 @import conf.switches.Switches._
 
+@if(OfflinePageView.isSwitchedOn) {
+    try {
+        var offlineViews = parseInt(localStorage.getItem('gu.offlineViews'));
+        for (var i = 0; i < offlineViews; i++) {
+            (new Image()).src = window.guardian.config.page.beaconUrl + '/count/offline-page-view.gif';
```

Comment:
yep, extending the metrics list in Diagnostics will allow the server to understand the beacon. As a rule, I've asked @sndrs to remove a metric, so we can stay neutral on number of metrics.